astro/src/content/quickstarts/quickstart-javascript-react-web.mdx

```diff
@@ -0,0 +1,198 @@
+
+## General Architecture
+
+While this sample application doesn't have login functionality without FusionAuth, a more typical integration will replace an existing login system with FusionAuth.
+
+In that case, the system might look like this before FusionAuth is introduced.
+
+<LoginBefore alt={"Request flow during login before FusionAuth"}/>
+
+The login flow will look like this after FusionAuth is introduced.
+
+<LoginAfterSdk alt={"Request flow during login after FusionAuth"}/>
+
+In general, you are introducing FusionAuth in order to normalize and consolidate user data. This helps make sure it is consistent and up-to-date as well as offloading your login security and functionality to FusionAuth.
+
+## Getting Started
+
+In this section, you'll get FusionAuth up and running and use React CLI to create a new application.
+
+### Clone the Code
+First off, grab the code from the repository and change into that directory.
+
+```shell
+git clone https://github.com/FusionAuth/fusionauth-quickstart-javascript-react-web.git
+cd fusionauth-quickstart-javascript-react-web
+```
+
+### Run FusionAuth via Docker
+In the root directory of the repo you'll find a Docker compose file (docker-compose.yml) and an environment
+variables configuration file (.env). Assuming you have Docker installed on your machine, you can stand up
+FusionAuth up on your machine with:
+
+```shell
+docker-compose up -d
+```
+
+This will start three containers, once each for FusionAuth, Postgres, and Elastic.
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart).
+When FusionAuth comes up for the first time, it will look at the `kickstart/kickstart.json` file and
+configure FusionAuth to a certain initial state.
+
+<Aside type="note">
+    If you ever want to reset the FusionAuth system, delete the volumes created by docker-compose by
+    executing `docker-compose down -v`, then re-run `docker-compose up -d`.
+</Aside>
+
+<p id="fusionauthconfig">FusionAuth will be configured with these settings</p>
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don’t need to.
+
+### Create a basic React application
+
+Now you are going to create a basic React application using the React CLI. While this section builds a simple React application, you can use the same configuration to integrate your existing React application with FusionAuth.
+
+```shell
+npx create-react-app changebank && cd changebank
```

Comment:
I've added a new aside note:
"It may ask you to install the `create-react-app` package, just confirm it. Further information can be found on the [Create React App Website](https://create-react-app.dev/docs/getting-started/)"